src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1008,6 +1008,26 @@ function ($a) {
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('transport')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->arrayNode('serializer')
+                                    ->canBeDisabled()
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('format')->defaultValue('json')->end()
+                                        ->arrayNode('context')
+                                            ->normalizeKeys(false)
+                                            ->useAttributeAsKey('name')
+                                            ->defaultValue(array())
+                                            ->prototype('variable')->end()
+                                        ->end()
+                                    ->end()
+                                ->end()
+                                ->scalarNode('encoder')->defaultValue('messenger.transport.symfony_serializer')->end()
```

Comment:
what about removing the `symdony_` here (and in the other places in this PR also)?

Reply:
I don't see either a benefit nor a downside 🙃

Reply:
Except that it's about Symfony's internal here so it might make sense to remove `symfony_`

Reply:
Removed 👍